### PR TITLE
Cu 86c0d0tp8 darklightdevice theme configuration

### PR DESCRIPTION
**Pull request by JacobCube**
Theme change is available between the options LIGHT/DARK/SYSTEM. The system is set by default, any change reconfigures the settings and saves it locally. The theme is changed in runtime, no application restart is required.

### CODE REVIEW

**Comment by JacobCube** [comment]
Task linked: [CU-86c0d0tp8 Dark/light/device theme configuration](https://app.clickup.com/t/86c0d0tp8)

File: composeApp/src/androidMain/kotlin/chat/enrichment/eu/MainActivity.kt
Comment (zeddosx) [inline composeApp/src/androidMain/kotlin/chat/enrichment/eu/MainActivity.kt:1]: new line na konci

File: composeApp/src/commonMain/composeResources/values/strings.xml [hunk 1]
```diff
@@ -33,13 +33,19 @@
   <string name="login_password_password_condition_empty">Is not empty, duh!</string>
   <string name="login_password_password_condition_0">Minimum 9 characters long.</string>
   <string name="login_password_password_condition_1">At least one algebra character.</string>
-  <string name="login_password_password_condition_2">At least very special character.</string>
+  <string name="login_password_password_condition_2">At least one very special character.</string>
   <string name="login_success_snackbar">Welcome, what can I get you?</string>
   <string name="login_success_snackbar_action">Just water, please.</string>
   <string name="login_password_action_go">Here I go!</string>
```
Comment (zeddosx) [inline composeApp/src/commonMain/composeResources/values/strings.xml:41]: tfuj

File: composeApp/src/commonMain/composeResources/values/strings.xml [hunk 2]
```diff
@@ -33,13 +33,19 @@
...
 
 
   <string name="account_dashboard_fcm">Your FCM token on this device is:</string>
+  <string name="account_dashboard_theme">Dark and darker</string>
+  <string name="account_dashboard_theme_light">Light</string>
+  <string name="account_dashboard_theme_dark">Dark</string>
+  <string name="account_dashboard_theme_device">System</string>
+
+  <string name="splash_loading_text">We are waking up the little people operating your application...</string>
```
Comment (zeddosx) [inline composeApp/src/commonMain/composeResources/values/strings.xml:49]: tfuj

File: composeApp/src/commonMain/kotlin/base/theme/AppThemeShapes.kt
Comment (zeddosx) [inline composeApp/src/commonMain/kotlin/base/theme/AppThemeShapes.kt:1]: new line na konci

File: composeApp/src/commonMain/kotlin/data/io/app/LocalSettings.kt
Comment (zeddosx) [inline composeApp/src/commonMain/kotlin/data/io/app/LocalSettings.kt:1]: new line na konci

File: composeApp/src/commonMain/kotlin/data/io/app/SettingsKeys.kt
Comment (zeddosx) [inline composeApp/src/commonMain/kotlin/data/io/app/SettingsKeys.kt:1]: new line na konci

File: composeApp/src/commonMain/kotlin/data/shared/SharedDataManager.kt
Comment (zeddosx) [inline composeApp/src/commonMain/kotlin/data/shared/SharedDataManager.kt:1]: new line na konci

File: shared/src/commonMain/kotlin/chat/enrichment/shared/ui/components/MultiChoiceSwitch.kt
Comment (zeddosx) [inline shared/src/commonMain/kotlin/chat/enrichment/shared/ui/components/MultiChoiceSwitch.kt:1]: new line na konci

**Review by zeddosx — APPROVED** [review]

**Pull request merged**